src/hotspot/cpu/aarch64/relocInfo_aarch64.hpp

```diff
@@ -31,8 +31,8 @@
   enum {
     // Relocations are byte-aligned.
     offset_unit        =  1,
-    // We don't use format().
-    format_width       =  0
+    // Must be at least 1 for RelocInfo::narrow_oop_in_const.
+    format_width       =  1
```

Comment:
Did you hit an issue with  = 0? Yes, it should be 1 for 64-bit. But I surprise nobody noticed this until now. How we handled embedded narrow_oop_in_const before?

Reply:
The assertion `assert((format & format_mask) == format, "wrong format");` fails in `relocInfo::check_format_and_offset()`. That comes from the call in `jvmciCodeInstaller.cpp` line 866. I don't know why we don't see this with regular Graal usage. Maybe it doesn't use that relocation type on AArch64?

Reply:
@dougxc Can you check why we did not hit this assert before?

BTW, I am fine with these changes.

Reply:
Most likely because we are not currently testing GraalVM on non-product JDK builds on AArch64.